README.md

```diff
@@ -283,89 +300,124 @@ cd test-network-function && ./test-network-function.test --help
+The 'General' tests are designed to test any commodity CNF running on OpenShift, and include specifications such as
+'Default' network connectivity.
 
-### Network Interfaces
+'CNF-specific' tests are designed to test some unique aspects of the CNF under test are behaving correctly.  This could
+include specifications such as issuing a `GET` request to a web server, or passing traffic through an IPSEC tunnel.
 
-* The annotation `test-network-function.com/defaultnetworkinterface` is the highest priority, and must contain a
-JSON-encoded string of the primary network interface for the pod. This must be explicitly set if needed. Examples can
-be seen in [cnf-certification-test-partner](https://github.com/test-network-function/cnf-certification-test-partner/local-test-infra/local-pod-under-test/local-partner-pod.yaml)
-* If the above is not present, the `k8s.v1.cni.cncf.io/networks-status` annotation is checked and the `"interface"` from
-the first entry found with `"default"=true` is used. This annotation is automatically managed in OpenShift but may not
-be present in K8s.
+'CNF-specific' test are yet to be defined.
 
-### Container Spec
+### General tests
 
-* Pods to be tested by the `container` spec are identified with the `test-network-function.com/container=target`
-label. Any value is permitted but `target` is used here for consistency with the `generic` spec.
-* If specific tests are to be run on the pod then they can be listed as a JSON-encoded list of strings under the
-`test-network-function.com/container_tests` annotation. If the annotation is not found then by default every
-group of CNF tests defined in `testconfigure.yml` will be run.
+Test in the "general" category belong to multiple suites that can be run in any combination as is
+appropriate for the CNF(s) under test. Test suites group tests by topic area:
 
-For example:
-```yaml
-...
-  labels:
-    test-network-function.com/container: target
-  annotations:
-    test-network-function.com/container_tests: "[\"PRIVILEGED_POD\",\"PRIVILEGED_ROLE\"]" # optional
-...
-```
+Suite|Test Spec Description|Minimum OpenShift Version
+---|---|---
+`access-control`|The access-control test suite is used to test  service account, namespace and cluster/pod role binding for the pods under test. It also tests the pods/containers configuration.|4.4.3
+`affiliated-certification`|The affiliated-certification test suite verifies that the pod under test and operator under test are certified by Redhat|4.4.3
+`diagnostic`|The diagnostic test suite is used to gather node information from an OpenShift cluster.  The diagnostic test suite should be run whenever generating a claim.json file.|4.4.3
+`lifecycle`| The lifecycle test suite verifies that the pod deployment, creation, shutdown and  survivability. |4.4.3
+`networking`|The generic test suite is used to test `Default` network connectivity between containers.|4.4.3
```

Comment:
This suite also does connectivity check with multus network. Take a look at the comment I put in doc.go and see if it can be used here.